systest/Makefile

```diff
@@ -41,22 +42,26 @@ docker:
 push:
 	docker push $(image_name)
 
+.PHONY: template
+template:
+	@echo smesher=`cat $(smesher_config)` >> $(tmpfile)
+	@echo poet=`cat $(poet_config)` >> $(tmpfile)
```

Comment:
i am removing a file after reading it in kubectl, see config subcommand
this is the best i could come up with. do you know how it can be done better?

Reply:
actually didnt notice, changing >> to > is neat. will use